### PR TITLE
add adx rating to adx indicator

### PR DESCRIPTION
**Pull request by ooples**
This is my first commit for adding adx rating to adx indicator. Don't see an issue number for it but it is in the backlog. Hopefully I'm doing this right

### CODE REVIEW

**Comment by DaveSkender** [comment]
Just a few tips initially:

- avoid using the `main` branch of your Fork as it will limit your ability to keep it in sync with the main repo; create a branch on your Fork, then work from there
- keep the PR checklist in the original Description (above), so we can track all the things we'll need to do to complete the work
- use the "draft" variant of the PR if you intend to have more updates
- requests a review (from me) when you're done making changes and are ready to do code review
- I can help in some areas by making commits on your Fork's branch or by just providing guidance, but let me know where and how you want help (e.g. which of the checklist items)

**Comment by DaveSkender** [comment]
Also, I have to convert the Project item into an Issue before it's a thing.  I've done that now.  See #693

**Comment by ooples** [comment]
I can add all of the missing indicators first and let me know if I need to do anything to fix this pull request

**Comment by DaveSkender** [comment]
> let me know if I need to do anything to fix this pull request

For this specific update, we're missing these checklist items:

- [ ] updated unit tests (and corresponding manual calculations in the spreadsheet, preferably)
- [ ] updated documentation
- [ ] code review

I can help with some of these, but please do let me know which areas you want help.

**Comment by ooples** [comment]
I'm personally unfamiliar with unit tests so not sure where to start on
that. Not sure where I need to go to update the documentation and not sure
what I need to do for the code review step

**Comment by DaveSkender** [comment]
> I'm personally unfamiliar with unit tests so not sure where to start on that. Not sure where I need to go to update the documentation and not sure what I need to do for the code review step

I can walk you through some of this, no problem.  Do you want to do some live pair-programming together or do you want me to just add the missing pieces so you can see them in the commit history?

**Comment by ooples** [comment]
I can virtually meet up later but right now my wife is dragging me away to
go snowshoe hiking with her

Franklin

**Comment by DaveSkender** [comment]
> I can virtually meet up later but right now my wife is dragging me away to go snowshoe hiking with her

Just shoot me an email or calendar invite to schedule.  I'll be busy with other stuff during the day too.

File: src/a-d/Adx/Adx.cs [hunk 1]
```diff
@@ -124,6 +124,10 @@ public static partial class Indicator
             {
                 adx = ((prevAdx * (lookbackPeriods - 1)) + dx) / lookbackPeriods;
                 result.Adx = adx;
+
+                double priorAdx = results.ElementAtOrDefault(index - lookbackPeriods).Adx.GetValueOrDefault();
```
Comment (DaveSkender) [inline src/a-d/Adx/Adx.cs:128]: Does this look okay?

**Review by DaveSkender — APPROVED** [review]

**Pull request merged**